lib/shopify_api/resources/fulfillment_order.rb

```diff
@@ -34,13 +34,52 @@ def close
       load_attributes_from_response(post(:close, {}, only_id))
     end
 
+    def fulfillment_request(fulfillment_order_line_items:, message:)
+      body = {
+        fulfillment_request: {
+          fulfillment_order_line_items: fulfillment_order_line_items,
+          message: message
+        }
+      }
+      keyed_fos = load_keyed_attributes_from_response(post(:fulfillment_request, {}, body.to_json))
+      if keyed_fos&.fetch('original_fulfillment_order', nil)&.attributes
+        load(keyed_fos['original_fulfillment_order'].attributes, false, true)
+      end
```

Comment:
What does this do?

Reply:
The caller code will look like `fulfillment_order.fulfillment_request(...)`. This code will use the attributes of the fulfillment order in the response body with hash key `original_fulfillment_order` to update the `fulfillment_order` that was used to call `.fulfillment_request`